packages/x-components/src/x-modules/identifier-results/components/identifier-result.vue

```diff
@@ -57,14 +57,14 @@
      * @public
      */
     protected get highlightedQueryHTML(): string {
-      const identifierValue = this.result.identifier.value;
+      const identifierValue = this.result.identifier?.value;
```

Comment:
I think we can move the `?? ''` (nullish coalescing operator) here:

```suggestion
      const identifierValue = this.result.identifier?.value ?? '';
```